components/SkilledImpact/elements/OurSkillImpactGroup.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import styles from '../../../constants/SkilledImpact/OrgSkilledImpactBody';
+import ShieldWithCheckMark from '../../../assets/jsicons/miscIcons/ShieldWithCheckMark';
+import ChevronBottom from '../../../assets/jsicons/miscIcons/ChevronBottom';
+import ChevronRight from '../../../assets/jsicons/miscIcons/ChevronRight';
+
+class OurSkillImpactGroup extends React.Component {
+  constructor(props) {
+    //TODO props edits and states
+    super(props);
+    this.state = {
+      skillExpand: true,
+    };
+  }
+
+  skillExpand = () =>{
+    this.setState({skillExpand: !this.state.skillExpand})
+  };
```

Comment:
**nit**: Let's rename this function to `toggleExpanded()` and the state variable to `expanded`.